web/src/components/tasks/RaffleBingoTask.tsx

```diff
@@ -0,0 +1,17 @@
+import { TaskHeader } from "@/components/TaskHeader";
+import { RaffleBingoPaginator } from "@/components/tasks/RaffleBingoPaginator";
+import { Task } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+interface Props {
+  task: Task;
+}
+
+export const RaffleBingoTask = (props: Props): ReactElement => {
+  return (
+    <div>
+      <TaskHeader task={props.task} />
+      <RaffleBingoPaginator task={props.task} />
```

Comment:
I know this is a pattern we follow other places in the application, but I'm not totally sold on shoving all of the task logic into the paginator. The only thing the task component is really doing here is adding a `TaskHeader`. Personally, I'd opt for either keeping all the logic in a single file (e.g. moving everything in `RaffleBingoPaginator` up one level into `RaffleBingoTask`), or divvying up logical responsibilities between the paginator component and the task component. 

This just feels like `RaffleBingoTask` doesn't have purpose.

Curious what your thoughts are here.

Reply:
➕

Reply:
@AnJuHyppolite Thoughts here?